bazel/linux/qemu.bzl

```diff
@@ -49,6 +49,7 @@ fi
 
 QEMU_FLAGS+=("-fsdev" "local,security_model=none,multidevs=remap,id=fsdev-fsOutputDir,path=$OUTPUT_DIR")
 QEMU_FLAGS+=("-device" "virtio-9p-pci,fsdev=fsdev-fsOutputDir,mount_tag=/dev/output_dir")
+KERNEL_FLAGS+=("OUTPUT_DIR=/tmp/output_dir")
```

Comment:
how does this go from /proc/cmdline to env variable? don't see anything in init-qemu-template.sh?

Reply:
According to https://www.kernel.org/doc/html/latest/admin-guide/kernel-parameters.html:
> parameters with ‘=’ go into init’s environment

Reply:
After thinking a bit about your earlier comment, I realized that although it sounds interesting to pass the environment variable through the kernel command line, it is not really needed here. The location of the OUTPUT_DIR in the guest can be chosen arbitrarily by the guest. There is no need for the host to know or control the placement.

It's good to know that we can pass environment variables from the host to the guest, but in this case it really doesn't give any benefit.

Reply:
makes sense, so remove it? good point.